CMakeLists.txt

```diff
@@ -120,10 +120,17 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/openarm-can.pc
 
 add_executable(motor-check setup/motor_check.cpp)
 target_link_libraries(motor-check openarm_can)
+install(TARGETS motor-check DESTINATION libexec/${PROJECT_NAME})
 
 # Add motor control example executable
 add_executable(openarm-demo examples/demo.cpp)
 target_link_libraries(openarm-demo openarm_can)
+install(TARGETS openarm-demo DESTINATION libexec/${PROJECT_NAME})
+
+# Install scripts
+install(PROGRAMS setup/change_baudrate.py setup/configure_socketcan.sh
+                 setup/configure_socketcan_4_arms.sh setup/set_zero.sh
+        DESTINATION libexec/${PROJECT_NAME})
```

Comment:
`${PROJECT_NAME}` will be `openarm_can`, but the path should be `libexec/openarm-can`.
How should we handle this?

Reply:
Fixed: https://github.com/enactic/openarm_can/compare/af43ad6947e797243b25920483b1d4d2bd8a5a7d..3e0df06cd067e1d044558d18a8f2d44bc48b0085